etc/a.gwf

```diff
@@ -213,6 +232,19 @@ propose_add_family=yes
 # proposed in the welcome page.
 propose_place_surname=yes
 
+# provide a list of names for the decomposition of places (comma separated)
+# other = ""; town = ""; township = ""; canton = ""; district = "";
+# county = ""; region = ""; country = ""
+#
+#place=autre, ville, agglomération, canton, district, conté, région, pays
+
+# Spécific à to the API (?? TODO document ??)
+#places_format=
```

Comment:
Should be removed. `place` already exists and has the same purpose.

Reply:
ok